src/locales/pt.json

```diff
@@ -0,0 +1,217 @@
+{
+  "Blockchain": {
+    "Mnemonic": "Mnemônico",
+    "Address": "Endereço",
+    "Wallet": "Carteira",
+    "Wallets": "Carteiras",
+    "Coins": "Moedas",
+    "Tokens": "Tokens",
+    "Balance": "Saldo",
+    "Tx hash": "Hash Tx",
+    "Gas": "Gas",
+    "Tax": "Imposto",
+    "Fee": "Taxa",
+    "Memo": "Memo",
+    "decimals": "decimais",
+    "Buy": "Comprar",
+    "Exchanges": "Exchanges",
```

Comment:
Not sure about the context. It could also be `trocas`, perhaps?

Reply:
Same as before, "trocas" would not reflect exactly the meaning, but maybe I can use "câmbios"
This is used in the context of exchange of currencies, I guess "Câmbios" could be used.

Reply:
I checked Binance pt_BR, and they use the word "Converter" for this. So I will translate it to "Conversões"